growth/too/tests/test_ref.py

```diff
@@ -0,0 +1,16 @@
+import astropy
+
+from .. import models
+from ..tasks import ref
+
+
+def test_refs():
+    data_rows = [(1, 2, 1, 1, 22.0),
+                 (1, 2, 1, 2, 23.0),
+                 (2, 2, 1, 2, 23.0)]
+    names = ('field', 'ccid', 'qid', 'fid', 'maglimit')
+    refstable = astropy.table.Table(rows=data_rows, names=names)
```

Comment:
Please use pytest's [monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html) or the Python standard library's [unittest.mock.patch](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) to mock up the TAP query so that you do not have to modify the function under test.

Reply:
Don't we want this functionality? Same with the observation table?

Reply:
What do you mean by "this functionality?"